src/test/resources/network/brightspots/rcv/test_data/convert_to_cdf_from_dominion/convert_to_cdf_from_dominion_config.json

```diff
@@ -68,7 +68,7 @@
     "decimalPlacesForVoteArithmetic": "4",
     "minimumVoteThreshold": "0",
     "maxSkippedRanksAllowed": "1",
-    "maxRankingsAllowed": "8",
+    "maxRankingsAllowed": "5",
```

Comment:
Curious why this was necessary?

Reply:
Just minimizes the number of changes to the expected CSV: without it, every line would have three additional commas for the three additional rankings.